mhkit/utils/__init__.py

```diff
@@ -1,5 +1,12 @@
+"""
+This module initializes and imports the essential utility functions for data 
+conversion, statistical analysis, caching, and event detection for the 
+MHKiT library.
+"""
+
 from .time_utils import matlab_to_datetime, excel_to_datetime, index_to_datetime
 from .stat_utils import (
+    _calculate_statistics,
```

Comment:
new local function created to reduce the number of local variables in `get_statistics`